x/ccv/consumer/keeper/relay.go

```diff
@@ -194,17 +189,28 @@ func (k Keeper) SendPendingSlashRequests(ctx sdk.Context) {
 	k.ClearPendingSlashRequests(ctx)
 }
 
+// OnAcknowledgementPacket handles acknowledgments for sent VSCMatured and Slash packets
 func (k Keeper) OnAcknowledgementPacket(ctx sdk.Context, packet channeltypes.Packet, ack channeltypes.Acknowledgement) error {
 	if err := ack.GetError(); err != "" {
-		// packet was sent to a non-established channel
-		if err != sdkerrors.Wrap(
-			channeltypes.ErrInvalidChannelState,
-			packet.DestinationChannel,
-		).Error() {
-			return fmt.Errorf(err)
+		// Reasons for ErrorAcknowledgment
+		//  - packet data could not be successfully decoded
+		//	- packet sent on a non-established channel
+		//  - the Slash packet was ill-formed (errors while handling it)
+		// None of these should ever happen.
```

Comment:
In this case, it would be good to add some logging or maybe panic at the end of the code so that information is provided to node operator that this is not normal and should be reported